src/Squirrel/DownloadManager.cs

```diff
@@ -587,6 +587,18 @@ private long GetDownloadedFileSize(string fileUrl)
             }
         }
 
+        /// <summary>
+        /// Gets the size of the file.
+        /// </summary>
+        /// <param name="filename">Name of the file.</param>
```

Comment:
Is it only a file name or path?

Reply:
It is path and I've changed the param to the "path". I copied it from somewhere. Short explanation why I think there was a filename is that the _FileInfo_ constructor takes "fileName" which is described as fully qualified file name. So, it is also path and it is quite confusing if you don't read the doc.